sdk/resourcemanager/azure-resourcemanager/src/test/java/com/azure/resourcemanager/ApplicationGatewayTests.java

```diff
@@ -92,18 +96,20 @@ public void testAppGatewaysInternalComplex() throws Exception {
      *
      * @throws Exception
      */
+    @Disabled("FIXME Priority is required in UrlPathMap")
     @Test
     public void testAppGatewaysPublicUrlPathBased() throws Exception {
         new TestApplicationGateway().new UrlPathBased(azureResourceManager.resourceGroups().manager().internalContext())
             .runTest(azureResourceManager.applicationGateways(), azureResourceManager.resourceGroups());
     }
 
+    @Disabled("WAF_V2 currently does not support private endpoint, and we do not want to have accessible public IP on VM. Enable it after WAF_V2 supports private endpoint.")
```

Comment:
should this have a link to a tracking github issue

Reply:
Sure. Issue logged https://github.com/Azure/azure-sdk-for-java/issues/36122